src/libraries/icubmod/usbCamera/linux/V4L_camera.cpp

```diff
@@ -432,13 +476,13 @@ bool V4L_camera::fromConfig(yarp::os::Searchable& config)
     switch(type)
     {
         case VOCAB_PIXEL_MONO:
-            param.pixelType = V4L2_PIX_FMT_GREY;
-            param.dst_image_size = param.width * param.height;
+            // Pixel type raw is the native one from the camera
+            param.pixelType = convertV4L_to_YARP_format(param.src_fmt.fmt.pix.pixelformat);
             break;
 
         case VOCAB_PIXEL_RGB:
+            // is variable param.pixelType really required??
```

Comment:
This variable it is necessary if you want to use `usbCamera` for getting RGB or RAW images, setting `capabilities` in `grabberDual`.